copyright

```diff
@@ -2825,3 +2825,361 @@ License: CC-BY-SA-3.0
  additional rights are deemed to be included in the License; this
  License is not intended to restrict the license of any rights under
  applicable law.
+
+License: GPL-2
+ 0. This License applies to any program or other work which contains
```

Comment:
1. GPL License is not allowed a modification. I doubt we can remove the copyright notice and Preamble.
2. GPL License explains what GPL is, but doesn't explain what you would publish something by; I suggest inserting the following text.

```
This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License version 2 as published by
the Free Software Foundation.
.
This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.
```

Reply:
GPL is under GPL so we can do with it what we want?

Reply:
GPL is not under GPL.

Reply:
> 1. GPL License is not allowed a modification. I doubt we can remove the copyright notice and Preamble.

quyykk added the notice and preamble, so this should be okay now.

>     2. GPL License explains what GPL is, but doesn't explain what you would publish something by; I suggest inserting the following text.

The text suggested here looks like the text that we typically put in the header of our source-files. I feel we don't need to include it here. (People will typically look at other source files anyway to see what they should add to their own files.)